src/main/java/com/blackducksoftware/integration/alert/provider/blackduck/BlackDuckProperties.java

```diff
@@ -23,198 +23,74 @@
-        this.blackDuckUrl = blackDuckUrl;
-    }
-
-    public Optional<Boolean> getBlackDuckTrustCertificate() {
-        // TODO in 3.0.0 we should consider changing the @Value annotations with the new branding names AND @Value will check the environment variables for us, so we wont need to do these extra checks
-        if (blackDuckTrustCertificate == null) {
-            return Optional.empty();
-        }
-        final String alwaysTrust = getEnvironmentVariable(AlertEnvironment.HUB_ALWAYS_TRUST_SERVER_CERTIFICATE);
-        if (blackDuckTrustCertificate) {
-            return Optional.ofNullable(blackDuckTrustCertificate);
-        }
-        if (StringUtils.isNotBlank(alwaysTrust)) {
-            return Optional.ofNullable(Boolean.parseBoolean(alwaysTrust));
+        if (StringUtils.isNotBlank(blackDuckUrl)) {
+            return Optional.of(blackDuckUrl);
         }
-        return Optional.of(false);
-    }
-
-    public void setBlackDuckTrustCertificate(final Boolean blackDuckTrustCertificate) {
-        this.blackDuckTrustCertificate = blackDuckTrustCertificate;
-    }
-
-    public Optional<String> getBlackDuckProxyHost() {
-        final String proxyHost = getEnvironmentVariable(AlertEnvironment.HUB_PROXY_HOST);
-        if (StringUtils.isNotBlank(blackDuckProxyHost)) {
-            return Optional.ofNullable(blackDuckProxyHost);
-        } else {
-            return Optional.ofNullable(proxyHost);
-        }
-    }
-
-    public void setBlackDuckProxyHost(final String blackDuckProxyHost) {
-        this.blackDuckProxyHost = blackDuckProxyHost;
+        return Optional.empty();
     }
 
-    public Optional<String> getBlackDuckProxyPort() {
-        final String proxyPort = getEnvironmentVariable(AlertEnvironment.HUB_PROXY_PORT);
-        if (StringUtils.isNotBlank(blackDuckProxyPort)) {
-            return Optional.ofNullable(blackDuckProxyPort);
-        } else {
-            return Optional.ofNullable(proxyPort);
+    public Optional<String> getPublicBlackDuckWebserverHost() {
+        if (StringUtils.isNotBlank(publicBlackDuckWebserverHost)) {
+            return Optional.of(publicBlackDuckWebserverHost);
         }
+        return Optional.empty();
     }
 
-    public void setBlackDuckProxyPort(final String blackDuckProxyPort) {
-        this.blackDuckProxyPort = blackDuckProxyPort;
-    }
-
-    public Optional<String> getBlackDuckProxyUsername() {
-        final String proxyUser = getEnvironmentVariable(AlertEnvironment.HUB_PROXY_USER);
-        if (StringUtils.isNotBlank(blackDuckProxyUsername)) {
-            return Optional.ofNullable(blackDuckProxyUsername);
-        } else {
-            return Optional.ofNullable(proxyUser);
+    public Optional<String> getPublicBlackDuckWebserverPort() {
+        if (StringUtils.isNotBlank(publicBlackDuckWebserverPort)) {
+            return Optional.of(publicBlackDuckWebserverPort);
         }
+        return Optional.empty();
     }
```

Comment:
This class could also use the method I mentioned above. Perhaps we want to make some sort of properties class to add these methods